Library/Homebrew/cmd/style.rb

```diff
@@ -35,6 +37,15 @@ def style
       ARGV.formulae.map(&:path)
     end
 
+    # `--staged`: detect staged files
+    if ARGV.include? "--staged"
+      staged = `git -C "#{HOMEBREW_REPOSITORY}" diff --cached --name-only`.split
```

Comment:
`Utils.popen_read` looks less grim. Just saying 🙈

Reply:
Are you suggesting something like this:
```
staged = Utils.popen_read("git", "-C", HOMEBREW_REPOSITORY.to_s, "diff", "--cached", "--name-only").split
```

Reply:
Yeah, basically. I don't know if Homebrew has an official policy on this & literally any maintainer can overrule me but I never miss an opportunity to get anal about backticks.